pkg/apis/nats/v1alpha2/cluster.go

```diff
@@ -201,9 +201,16 @@ type RemoteGatewayOpts struct {
 	URL string `json:"url,omitempty"`
 }
 
+// LeafNodeRemote is the URL for remote NATS system.
+type LeafNodeRemote struct {
+	URL         string `json:"url,omitempty"`
```

Comment:
I think we should probably also support the array version:
```
URL         []string `json:"urls,omitempty"`
```

Reply:
Is this possible NATS config?
```
leafnodes {
    remotes = [
        {
          url: ["nats-leaf://s3cr3t@foo", "nats-leaf://s3cr3t@foo"]
        },
    ]
}
```

Reply:
yes also possible

Reply:
Ah, okay. Sounds good. :+1: